packages/eslint-plugin/src/rules/no-unnecessary-parameter-property-assignment.ts

```diff
@@ -0,0 +1,70 @@
+import type { TSESTree } from '@typescript-eslint/utils';
+import { AST_NODE_TYPES } from '@typescript-eslint/utils';
+
+import * as util from '../util';
+
+export const messageId = 'noUnnecessaryParameterPropertyAssignment';
```

Comment:
[Style] We don't generally do this. We normally go for small clean message IDs. The practice of storing them in variables doesn't scale well to rules that have many messages.

```suggestion
```